src/server/__mocks__/mockESData/mockNestedAggs.js

```diff
@@ -58,7 +59,7 @@ const mockNestedAggs = () => {
             },
             {
               key: {
-                'visits.visit_label': 'no data',
```

Comment:
why is this behavior changing? this should retured the value of `missing_data_alias` from the guppy config file, if not configured then the default value should be `'no data'`

Reply:
ES does not support missing configuration anymore. We need to use different way {missing_bucket: true,  order: 'desc'} to configure the field with missing data. 
This part will handle the missing data
https://github.com/uc-cdis/guppy/pull/200/files#diff-5db64ea5481e180cbf163669ddb74578c387c32bafcbb7a4dccc8d57c1dc1102R685-R689